src/layout.rs

```diff
@@ -278,6 +392,59 @@ impl Layout {
                     self.do_action(action, coord, delay);
                 }
             }
+            Sequence { events } => {
+                // Copy the first chunk of `SequenceEvent`s into the `sequenced` ArrayDeque
+                // ...we'll cover the remaining chunks as we drain them from `sequenced`
+                let chunk_length = self.sequenced.capacity() - 1; // -1: Keep a slot for Continue()
+                if let Some(chunk) = events.chunks(chunk_length).next() {
```

Comment:
My proposition will simplify this.

And what if a sequence is executed during the execution of another sequence? It need a test.

Reply:
Yeah, multiple simultaneous sequences is an edge case in the TODO list.  The way it works right now is that if you kick off a sequence while another one is already running it ends the current sequence and starts the new one.  I made the `sequenced` `ArrayDeque` 16 items long originally so I could have multiple simultaneous sequences running but the way the logic works right now that can't work.  I had planned to use the length of `sequenced` divided by four so you could have up to four simultaneous sequences going on but the amount of logic that would need to change for that to happen was too much (for the current state of the PR--can't have everything at the start of these things).

Reply:
You can document the actual behavior, and add an issue.